src/rp2_common/pico_audio_i2s/audio_i2s.pio

```diff
@@ -54,8 +73,13 @@ static inline void audio_i2s_program_init(PIO pio, uint sm, uint offset, uint da
 
     pio_sm_init(pio, sm, offset, &sm_config);
 
-    uint pin_mask = (1u << data_pin) | (3u << clock_pin_base);
+#if PICO_PIO_USE_GPIO_BASE
+    uint64_t pin_mask = (1ull << data_pin) | (3ull << clock_pin_base);
+    pio_sm_set_pindirs_with_mask64(pio, sm, pin_mask, pin_mask);
+#else
+    uint32_t pin_mask = (1ull << data_pin) | (3ull << clock_pin_base);
```

Comment:
Should the code in this `#else` part be using `(1u << data_pin) | (3u << clock_pin_base)` as it was originally, rather than using `ull` ?

Reply:
oh, yes. I'll fix it tomorrow.